Didi/entry address comments (#1007)
telerik/maui-docs: demos-and-sample-apps/overview.md

* [SDKBrowser App](#sdkbrowser-app)
* [Crypto Tracker App](#crypto-tracker-app)

> > With **Telerik .NET MAUI 7.1.0** version we have added support for .NET 9.0 You can easily switch the .NET version to .NET 8.0 or .NET 9.0 to build the apps. 

Reviewer: ```suggestion
> Telerik UI for .NET MAUI version 7.1.0 adds support for .NET 9.0. You can easily switch the .NET version to .NET 8.0 or .NET 9.0 to build the apps. 
```
